inc/taxonomies-functions.php

```diff
@@ -2243,3 +2248,96 @@ function taxopress_show_all_cpt_in_archive_result($request_tax){
 
             return $status;
 }
+
+/* Show taxonomy filter on post list */
+function taxopress_filter_dropdown( $taxonomy, $show_filter ) {
+
+    $show_filter   = get_taxopress_disp_boolean( $show_filter );
+                                        
+    if ( $show_filter == true ) {
+
+        wp_dropdown_categories(
+            array(
+                'show_option_all' => sprintf( __( 'All %s', 'simple-tags' ), $taxonomy->label ),
+                'orderby'         => 'name',
+                'order'           => 'ASC',
+                'hide_empty'      => false,
+                'hide_if_empty'   => true,
+                'selected'        => filter_input( INPUT_GET, $taxonomy->query_var, FILTER_SANITIZE_STRING ),
+                'hierarchical'    => true,
+                'name'            => $taxonomy->query_var,
+                'taxonomy'        => $taxonomy->name,
+                'value_field'     => 'slug',
+            )
+        );
+
+    }
+    
+}
+
+function taxopress_get_dropdown(){
+
+    global $pagenow;
+
+    if ( is_admin() ) {
+
+        $type = 'post';
+
+        if (isset($_GET['post_type'])) {
+
+            $type = $_GET['post_type'];
```

Comment:
All $_GET, $_REQUEST and $_POST input variable should be sanitized when been used. Kindly have this on mind when working on user provided input as they can easily be manipulated which can lead to security issue.

So, sanitize the `$type = $_GET['post_type'];`

Reply:
Hi @ojopaul I updated the code, thank you!